site/_includes/layouts/base.njk

```diff
@@ -17,12 +17,12 @@
     <link rel="preload" as="font" crossorigin href="/fonts/google-sans-v2003/regular/latin.woff2">
     <link rel="preload" as="font" crossorigin href="/fonts/google-sans-v2003/medium/latin.woff2">
 
-    
+
     {% block css %}{% endblock %}
 
     <script type="module" src="{{ helpers.hashForProd('/js/main.js') }}"></script>
-    {% if process.env.NODE_ENV === 'production' %}
-      <script async src="https://www.google-analytics.com/analytics.js"></script>
+    {% if process.env.NODE_ENV !== 'production' %}
```

Comment:
Reminder to myself - revert this change (which is only there for testing) before merging

Reply:
```suggestion
    {% if process.env.NODE_ENV === 'production' %}
```